subsys/net/lib/openthread/platform/settings.c

```diff
@@ -0,0 +1,313 @@
+/*
+ * Copyright (c) 2019 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <stdio.h>
```

Comment:
Typically in the networking subsys `snprintk()` is used, so if we use `snprintk()` instead of `snprintf()`, then this include could be removed.

Reply:
Thanks for the tip, fixed.